### PR TITLE
Updates to av overview templates

### PR DESCRIPTION
**Pull request by jdtoombs**
- update legend icons and colours
- fix syntax errors in template html
- make fixed width for templates to 595 px
- fix bugs where icons were not displaying

DB Migration 1.0.141

### CODE REVIEW

File: libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+"body" = '
+@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using System
+@using System.Linq
+@using TNO.Entities
+<style>
+a { text-decoration:none; }
+.video-icon { fill: #702828; height: 20px; width: 20px; }
+.transcript-icon { fill: #047A8C; height: 20px; width: 20px;}
+.icon { float: right; }
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql:14]: float as flex box will not work in email

File: libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+.main-body {
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql:15]: locked width

File: libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  width: 595px;
+  }
+</style>
+<div class="main-body">
+<div><p style="background-color: #FFF7E1; color: #876503; text-align: center; font-size: 16px;	font-weight: 700; line-height: 110%; letter-spacing: 0.08px; padding: 10px 0px; margin: 6px 0px 20px 0px;">This TNO product is intended only for the use of the person to whom it is addressed. Please do not forward or redistribute.</p></div>
+<div style="padding-top: 10px; border-radius:25px; background-color:rgba(255,255,255, .25)">
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql:21]: This was missing a closing quotation

File: libs/net/dal/Migrations/1.0.138/Up/PostUp/01-WeekendTemplateUpdate.sql [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  td.head {
+    font-weight: bold;
+  }
+  .small {
+    text-align: center;
+    width: 50px;
+  }
+
+  .main-body {
+   width: 595px;
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.138/Up/PostUp/01-WeekendTemplateUpdate.sql:38]: fixed width

File: libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                </a>
+              }
+              @if (item.Content.ContentType.ToString() == "AudioVideo" && item.Content.Summary.Length > 0 && item.Content.IsApproved)
+              {
+                <a class="icon" href="@($"{ViewContentUrl}{item.ContentId}")">
+<svg style="float: right" class="transcript-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M278.5 215.6L23 471c-9.4 9.4-9.4 24.6 0 33.9s24.6 9.4 33.9 0l57-57h68c49.7 0 97.9-14.4 139-41c11.1-7.2 5.5-23-7.8-23c-5.1 0-9.2-4.1-9.2-9.2c0-4.1 2.7-7.6 6.5-8.8l81-24.3c2.5-.8 4.8-2.1 6.7-4l22.4-22.4c10.1-10.1 2.9-27.3-11.3-27.3l-32.2 0c-5.1 0-9.2-4.1-9.2-9.2c0-4.1 2.7-7.6 6.5-8.8l112-33.6c4-1.2 7.4-3.9 9.3-7.7C506.4 207.6 512 184.1 512 160c0-41-16.3-80.3-45.3-109.3l-5.5-5.5C432.3 16.3 393 0 352 0s-80.3 16.3-109.3 45.3L139 149C91 197 64 262.1 64 330v55.3L253.6 195.8c6.2-6.2 16.4-6.2 22.6 0c5.4 5.4 6.1 13.6 2.2 19.8z"/></svg>
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.138/Up/PostUp/00-WeekdayTemplateUpdate.sql:88]: float as flex will not work

**Review by Fosol — COMMENTED** [review]
I was recently reviewing the existing templates for the evening overview.  The two were not the same for some reason.  It worries me to know which is the valid format.  Can you please confirm that this update to both of them is correct with Bobbi?

**Comment by Fosol** [comment]
@jdtoombs you should also update the related files we use in the UI related to these templates.
`/app/editor/src/features/admin/report-templates/constants`
`/app/editor/src/features/admin/report-templates/templates`

**Review by Fosol — APPROVED** [review]

**Pull request merged**